app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -81,6 +80,8 @@ def verify_credentials(auth_type = nil, options = {})
          RestClient::ResourceNotFound,
          RestClient::InternalServerError => err
     raise MiqException::MiqUnreachableError, err.message, err.backtrace
+  rescue Hawkular::BaseClient::HawkularException => err
+    raise MiqException::MiqInvalidCredentialsError, "HTTP status #{err.status_code}", err.backtrace
```

Comment:
We should probably have the message too. To be consistent with the existing error handling I would prefer:
```raise MiqException::MiqInvalidCredentialsError, err.status_code, err.message, err.backtrace```
(and make sure the error propagates as expected)